profiles/arch/base/use.mask

```diff
@@ -37,6 +37,9 @@ oci8
 cuda
 nvenc
 
+# cuDNN only works on amd64
```

Comment:
Can I add
`# Paul Zander <negril.nx+gentoo@gmail.com> (2023-12-10)`
here too?

Reply:
Sure. I just mirrored the cuda stuff and my prior HIP changes.